io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess.test/src/eu/esdihumboldt/hale/io/jdbc/msaccess/reader/MsAccessSchemaReaderTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.jdbc.msaccess.reader;
```

Comment:
Your test project should be added to the test product (and any dependencies so the product validates). Otherwise the tests are not executed within the CI.